eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.8.0.0
 
+* Add `ProposalProcedure` deposit check against "govActionDeposit" `PParams` #3669
```

Comment:
It is always good to actually add to the changelog the name of the function, type, type constructor, etc. that is being added or changed.

```suggestion
* Add `ProposalDepositIncorrect` predicate failure, that is produced when `ProposalProcedure` deposit does not match `"govActionDeposit"` from `PParams` #3669
```

Reply:
Noted! I don't know what I was thinking :bow: Thank you!